recipes-tag/grid-strategy/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "grid-strategy" %}
+{% set version = "0.0.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "d8db1c12e6f33eb55ba56a5b19f4848aae862a9a343fea422a353c46ab6ccc23" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
```

Comment:
This requires Matplotlib as well

Reply:
I see. It's not in `setup.py` for some reason.

Reply:
Done.

Reply:
This comment made me curious. It's [in their `setup.cfg`](https://github.com/matplotlib/grid-strategy/blob/db12cdf788f6ddca7cec231753b2014437942db3/setup.cfg#L26-L28) instead. I have not seen that approach before. I suppose the advantage is that INI is more descriptive and easier to externally parse than Python.

Reply:
Yeah, interesting, thanks for digging into the details, @danielballan! I also see there is a `numpy` dependency, and it's 3.6+ only. I'll make the corresponding changes to the recipe. Stay tuned.